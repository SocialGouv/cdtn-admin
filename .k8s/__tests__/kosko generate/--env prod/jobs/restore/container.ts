//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);

test("should restore from dev to preprod", async () => {
  expect(
    await getEnvManifests("prod", "jobs/restore/container", {
      ...project("cdtn-admin").prod,
      TRIGGER: "UPDATE_PREPROD",
    })
  ).toMatchSnapshot();
});

test("should restore from dev to prod", async () => {
  expect(
    await getEnvManifests("prod", "jobs/restore/container", {
      ...project("cdtn-admin").prod,
      TRIGGER: "UPDATE_PROD",
    })
  ).toMatchSnapshot();
});
