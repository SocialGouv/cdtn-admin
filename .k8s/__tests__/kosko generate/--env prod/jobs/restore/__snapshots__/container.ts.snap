// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should restore from dev to preprod 1`] = `
"---
apiVersion: batch/v1
kind: Job
metadata:
  annotations:
    kapp.k14s.io/update-strategy: skip
    app.gitlab.com/app: socialgouv-cdtn-admin
    app.gitlab.com/env: prod
    app.gitlab.com/env.name: prod
  labels:
    component: cdtn-admin
    application: cdtn-admin
    owner: cdtn-admin
    team: cdtn-admin
  name: restore-container-cdtn-files-to-dev-cdtn-preprod-v1-2-3-65ps4o
  namespace: cdtn-admin
spec:
  backoffLimit: 0
  template:
    metadata:
      annotations:
        kapp.k14s.io/deploy-logs: for-new-or-existing
    spec:
      containers:
        - command:
            - sh
            - '-c'
            - >

              [ ! -z $SOURCE_CONTAINER ] || (echo \\"No SOURCE_CONTAINER\\"; exit 1)

              [ ! -z $SOURCE_ACCOUNT_NAME ] || (echo \\"No SOURCE_ACCOUNT_NAME\\";
              exit 1)

              [ ! -z $SOURCE_ACCOUNT_KEY ] || (echo \\"No SOURCE_ACCOUNT_KEY\\";
              exit 1)

              [ ! -z $DESTINATION_CONTAINER ] || (echo \\"No
              DESTINATION_CONTAINER\\"; exit 1)

              [ ! -z $DESTINATION_ACCOUNT_NAME ] || (echo \\"No
              DESTINATION_ACCOUNT_NAME\\"; exit 1)

              [ ! -z $DESTINATION_ACCOUNT_KEY ] || (echo \\"No
              DESTINATION_ACCOUNT_KEY\\"; exit 1)

              echo \\"starting restore container $SOURCE_CONTAINER into
              $DESTINATION_CONTAINER\\"

              # copy container

              az storage copy -r   --source-account-name
              \\"$SOURCE_ACCOUNT_NAME\\"   --source-account-key
              \\"$SOURCE_ACCOUNT_KEY\\"   --source-container \\"$SOURCE_CONTAINER\\"  
              --account-name \\"$DESTINATION_ACCOUNT_NAME\\"   --account-key
              \\"$DESTINATION_ACCOUNT_KEY\\"   --destination-container
              \\"$DESTINATION_CONTAINER\\"

              # get original permission

              PUBLIC_ACCESS=\`az storage container show-permission  
              --account-name \\"$SOURCE_ACCOUNT_NAME\\"   --account-key
              \\"$SOURCE_ACCOUNT_KEY\\"   --name $SOURCE_CONTAINER | jq -r
              \\".publicAccess\\"\`

              # set original permission

              az storage container set-permission   --name
              $DESTINATION_CONTAINER   --account-name
              \\"$DESTINATION_ACCOUNT_NAME\\"   --account-key
              \\"$DESTINATION_ACCOUNT_KEY\\"   --public-access $PUBLIC_ACCESS
          env:
            - name: SOURCE_CONTAINER
              value: cdtn
            - name: SOURCE_ACCOUNT_NAME
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountname
                  name: azure-cdtnadmindev-volume
            - name: SOURCE_ACCOUNT_KEY
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountkey
                  name: azure-cdtnadmindev-volume
            - name: DESTINATION_CONTAINER
              value: cdtn-preprod
            - name: DESTINATION_ACCOUNT_NAME
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountname
                  name: azure-cdtnadmindev-volume
            - name: DESTINATION_ACCOUNT_KEY
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountkey
                  name: azure-cdtnadmindev-volume
          image: mcr.microsoft.com/azure-cli:2.15.1
          imagePullPolicy: IfNotPresent
          name: restore-db
          resources:
            limits:
              cpu: 300m
              memory: 512Mi
            requests:
              cpu: 50m
              memory: 64Mi
      restartPolicy: Never
  ttlSecondsAfterFinished: 86400
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-cdtnadmindev-volume
  namespace: cdtn-admin
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgAFpMndUw7Mo4QLdKCaW+mOLMauIXGu+cHv9F4sEbsVBnGy/oXmuBVU6hNiXJ/58JHSXSVsnpp8qYu9GhAHdQmN5crWmWDqfqgXd/03iH80QFyR+0PMMQZwsAqsnbzoox8SlghhwiwazdCB2y0C0VsxCd7sSC1uD+t9BUlIX1vRRTosUrZwD3SfKQI3otEXze+/fyo6pEnxMfvTs/fPZtIn4ixt8JL3hrLEDpavjLScIf1VgPlxG7+LC2CYcAeQ+k5hmtlLIFrjYRwlOEhrRNDCbgjEtOzVPuT3cCYAfuC8rFa4QW0kSRH8SRd4azQfNKfZNsUH8/bCq91h2zDRNpVSCm/ZWSQZYFPhmgRMlvqUNEqBAURQUUE1FwkNohr02jBeISrm4PW/7cSdoIhJ5Q7kXle2PzmC+Cv1n9y67OlCtxHSqOik+rCH2ryqoY+sKgoAISVvVWyhAf5/1VWqKudQDkJpbAyV7zh9orRbgKohtjAxFRdWK/+YplEDC/xWI8FnZd26PS5Cs/hRByDf5EVmDFNir64lPJ5kHYyLnFxF50RD9UQRPJ6qGRLO3gEYKRfle6Vox00lZ9y/ck2MAJo6/ZChksclBQYMC++Ky69zlQhQu1WzQGxV5iQRTDwQSHdzcHvEF7HEM390Fq6bglib00VOVx7wYFS4TgcrEgwKYIt1bkvtea204W8/6yn5WYZN8OiLElDmVy+0xcHVUwokdPlGEkR6YepI2dircEj89WzFELN7M2J4wbaEqkVODnEe8itGl+yhXiYDpsPzVnGulRwdNhIfWbf8Hq9ILY8GbXO6LdG4dJhL
    azurestorageaccountname: >-
      AgBm39+zb0dIlbRiM17Cg++XLr04EzShAbwHORvltWJE1HWGvCEMIFuEFjD2NOCuScOGK6b2abqLOadoDnENC+3HKwjwNQKRADL/TCX0bW4DMCJJ4RabMXalaaq19BSa1lN2yMgW1TO+lwVibiIxIpEbbYGpbL+U07Qraf5cNi9O+2Pc6hXmVWDoTi2Op9pEa0aYhKq1wIxB4B4kowH5RjsskdlXy2Gf2pf5QL4EFVBae8MMwH2INsJc/R90tuIGSwgr+FQ13iD1E3sEGi1VBqAcIRwA99UVUbsDVQc6Li/ENbJuS5JR9wPRqfvr+D5O5n+iRcYbDabo7R+bLIm0IdgAmIvREGmAHmq81Z6txWiJeUIsRZ3BldN9JGnU1fUpmA78TVhfpq42zy6q2EDvjIc91CL0jIJNI2mGljha2T1/gy2vqnIbTDj0zHBVajPEO+26vlUSd9zmDETf4Km3V2tYblUIiDuZwqH2hiiQz4lf1UYqF2sS5XlxcXBBP98YaLHNt0S5N/k1avH/XuaJbPyMyU9ZKE1IXWYTLnoMVnPM+/lD0J9xfeZJ3gLeLtjilARlk6FdzDx5W7IH4u9sdFEktnKq8NXRhkeYXb3CHtSjxYy1PfF6tSOCP9DIaZMcK2Z9Vqivis3+ZbFqxe9vWU92xzRg5ZsPr+AkXHnPOLKvt/SCnjRLT5taJVCBfvOSEXkg3tDKk2DWFczzpHA=
  template:
    metadata:
      annotations: {}
      name: azure-cdtnadmindev-volume
    type: Opaque
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-cdtnadminprod-volume
  namespace: cdtn-admin
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgArMu861qm2F7g/QpkOUqz4BAM+AIwuAXsE2PpJMNKKmZz/Pbyg3oIMfdcqg8e3rss7PpddLbZv+TYFCC8HuVIrQ9u5gA1t2HN0qjDGBsDyWr7cF2I+eB71JN287MoOeG90hF+TfkHh3bONZTDEwOT+sYs6XSDm4EsXEE7kkUk0yJjUA0s5uXZYI33dteRWPYMp7KaG73RWl6CW0/woKwnX5ml4d69g8cmAkf345Xe5knqGTh8wt2idJjEnjgF8j3WqCdbD1AUkqnovQ5igr36E/j8+B6GyNZtQVEqSyNZT16o973IpkoOs3ZUk5kAi9TlAxHIdLIUDqmnKr29apu/UGTdOGr0zqUbnJyawh+aCUSzLP3D/CKQFtcGNvs+FqUciOlRt1FoU25GuDMEH7ol3ic5+45jOMeJJNPwhOHxynJvAMQCnJAqGYMQefj4o5LnfRBYpS4CumMGhzuBvfHkVnrYj69d+gJ3YEpJ5aKBu9tCAAgyl9g8KzRTfLPCiSqxKfV7uQoNHilluWdHW8TQssHSzi1+K3WVXhRVTTV0NjQoEt9ZkPb7/iOns8IW5SucSJUABDf7U8C3aG+t0zymjMhpkpsnq/JPK2K4pkO8Qa0R6GSO6YJKVT9sKCYTfD3Mrr3FnROaThouPmQmCFfzVXErgdnAaBJUrG7F5vppguDA+obT9GH13zrurn+bEnvZfgi2oIutV/CMHijYS3tTA0OIEdHZRUb1DHU+efX4YuJc4qF2W117ZgUjv9luxAIFcPRXpkDKeUv1gGYAuKIMatkmLJ6END8iakKgb2NpfsNna8a1MSjAe
    azurestorageaccountname: >-
      AgAfpzqFtomji8nwYM6g5mbXXDAA8REQtfJakS059Vy2KkUbZ1okjVQ5mFN61ODTcBknR/dbncVRsl9n/JoEqLfN1wSp58QOBTFLFXiMNeLACEEIYg/Ko6hZVsUB6xs6x0Ma75ngDr+BcAtUCX9g7qqd4Lpw+7lWNdDrO4yT7BLdbV1WvoHA8YQkSyqZ/w9/lugYwc63mmf5PsUz3JVhHMwN+esOvTOcDqxFwpSR2v3z/o7wgmm3CLSg06LGSK5rAppetk/ojfeWZqWk9nA6It/BQ2x1nlku/2wU4HjO7H1+syPsvftElu4Bw4vcX8SFY5+2oGplwekThIkH/V+YQvsq1mI0y45ZOh8QfwfNJoPNjvEFGXpZX8eKriUAeesMC0y7CRMeSBVXdrmQFe9yjViFsh1gX5G5Jl7bFKMhZORZ/DAvVUfXt4zC/V+y3uo6QrFEp8Ywm/uYO5KwhiGU2B/8gYu3oQUuhjl8M+s+y8jkF+WeCTPe0lsxCYlxQ/BB9e1x86j+ulPtUEvlA4Bp/AMlKUomdLDuX8Mddtp2T85QB+3KuTtBQlWLRqjywhEZjKmourH99JOF2Buh9OX/tGg7S5ZfuUdwHqQh5/dX1S2/VFAJFdFOpt0HEzuiuxTw2l1rw5SYqmZuHRrbVIHgTyxRpUcJXQ+dep6MAuklR42sxynraa620atGW1MdHALd6U90DG8H482QS97v9lH3
  template:
    metadata:
      annotations: {}
      name: azure-cdtnadminprod-volume
    type: Opaque
"
`;

exports[`should restore from dev to prod 1`] = `
"---
apiVersion: batch/v1
kind: Job
metadata:
  annotations:
    kapp.k14s.io/update-strategy: skip
    app.gitlab.com/app: socialgouv-cdtn-admin
    app.gitlab.com/env: prod
    app.gitlab.com/env.name: prod
  labels:
    component: cdtn-admin
    application: cdtn-admin
    owner: cdtn-admin
    team: cdtn-admin
  name: restore-container-cdtn-files-to-prod-cdtn-v1-2-3-49ptwk
  namespace: cdtn-admin
spec:
  backoffLimit: 0
  template:
    metadata:
      annotations:
        kapp.k14s.io/deploy-logs: for-new-or-existing
    spec:
      containers:
        - command:
            - sh
            - '-c'
            - >

              [ ! -z $SOURCE_CONTAINER ] || (echo \\"No SOURCE_CONTAINER\\"; exit 1)

              [ ! -z $SOURCE_ACCOUNT_NAME ] || (echo \\"No SOURCE_ACCOUNT_NAME\\";
              exit 1)

              [ ! -z $SOURCE_ACCOUNT_KEY ] || (echo \\"No SOURCE_ACCOUNT_KEY\\";
              exit 1)

              [ ! -z $DESTINATION_CONTAINER ] || (echo \\"No
              DESTINATION_CONTAINER\\"; exit 1)

              [ ! -z $DESTINATION_ACCOUNT_NAME ] || (echo \\"No
              DESTINATION_ACCOUNT_NAME\\"; exit 1)

              [ ! -z $DESTINATION_ACCOUNT_KEY ] || (echo \\"No
              DESTINATION_ACCOUNT_KEY\\"; exit 1)

              echo \\"starting restore container $SOURCE_CONTAINER into
              $DESTINATION_CONTAINER\\"

              # copy container

              az storage copy -r   --source-account-name
              \\"$SOURCE_ACCOUNT_NAME\\"   --source-account-key
              \\"$SOURCE_ACCOUNT_KEY\\"   --source-container \\"$SOURCE_CONTAINER\\"  
              --account-name \\"$DESTINATION_ACCOUNT_NAME\\"   --account-key
              \\"$DESTINATION_ACCOUNT_KEY\\"   --destination-container
              \\"$DESTINATION_CONTAINER\\"

              # get original permission

              PUBLIC_ACCESS=\`az storage container show-permission  
              --account-name \\"$SOURCE_ACCOUNT_NAME\\"   --account-key
              \\"$SOURCE_ACCOUNT_KEY\\"   --name $SOURCE_CONTAINER | jq -r
              \\".publicAccess\\"\`

              # set original permission

              az storage container set-permission   --name
              $DESTINATION_CONTAINER   --account-name
              \\"$DESTINATION_ACCOUNT_NAME\\"   --account-key
              \\"$DESTINATION_ACCOUNT_KEY\\"   --public-access $PUBLIC_ACCESS
          env:
            - name: SOURCE_CONTAINER
              value: cdtn
            - name: SOURCE_ACCOUNT_NAME
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountname
                  name: azure-cdtnadmindev-volume
            - name: SOURCE_ACCOUNT_KEY
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountkey
                  name: azure-cdtnadmindev-volume
            - name: DESTINATION_CONTAINER
              value: cdtn
            - name: DESTINATION_ACCOUNT_NAME
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountname
                  name: azure-cdtnadminprod-volume
            - name: DESTINATION_ACCOUNT_KEY
              valueFrom:
                secretKeyRef:
                  key: azurestorageaccountkey
                  name: azure-cdtnadminprod-volume
          image: mcr.microsoft.com/azure-cli:2.15.1
          imagePullPolicy: IfNotPresent
          name: restore-db
          resources:
            limits:
              cpu: 300m
              memory: 512Mi
            requests:
              cpu: 50m
              memory: 64Mi
      restartPolicy: Never
  ttlSecondsAfterFinished: 86400
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-cdtnadmindev-volume
  namespace: cdtn-admin
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgAFpMndUw7Mo4QLdKCaW+mOLMauIXGu+cHv9F4sEbsVBnGy/oXmuBVU6hNiXJ/58JHSXSVsnpp8qYu9GhAHdQmN5crWmWDqfqgXd/03iH80QFyR+0PMMQZwsAqsnbzoox8SlghhwiwazdCB2y0C0VsxCd7sSC1uD+t9BUlIX1vRRTosUrZwD3SfKQI3otEXze+/fyo6pEnxMfvTs/fPZtIn4ixt8JL3hrLEDpavjLScIf1VgPlxG7+LC2CYcAeQ+k5hmtlLIFrjYRwlOEhrRNDCbgjEtOzVPuT3cCYAfuC8rFa4QW0kSRH8SRd4azQfNKfZNsUH8/bCq91h2zDRNpVSCm/ZWSQZYFPhmgRMlvqUNEqBAURQUUE1FwkNohr02jBeISrm4PW/7cSdoIhJ5Q7kXle2PzmC+Cv1n9y67OlCtxHSqOik+rCH2ryqoY+sKgoAISVvVWyhAf5/1VWqKudQDkJpbAyV7zh9orRbgKohtjAxFRdWK/+YplEDC/xWI8FnZd26PS5Cs/hRByDf5EVmDFNir64lPJ5kHYyLnFxF50RD9UQRPJ6qGRLO3gEYKRfle6Vox00lZ9y/ck2MAJo6/ZChksclBQYMC++Ky69zlQhQu1WzQGxV5iQRTDwQSHdzcHvEF7HEM390Fq6bglib00VOVx7wYFS4TgcrEgwKYIt1bkvtea204W8/6yn5WYZN8OiLElDmVy+0xcHVUwokdPlGEkR6YepI2dircEj89WzFELN7M2J4wbaEqkVODnEe8itGl+yhXiYDpsPzVnGulRwdNhIfWbf8Hq9ILY8GbXO6LdG4dJhL
    azurestorageaccountname: >-
      AgBm39+zb0dIlbRiM17Cg++XLr04EzShAbwHORvltWJE1HWGvCEMIFuEFjD2NOCuScOGK6b2abqLOadoDnENC+3HKwjwNQKRADL/TCX0bW4DMCJJ4RabMXalaaq19BSa1lN2yMgW1TO+lwVibiIxIpEbbYGpbL+U07Qraf5cNi9O+2Pc6hXmVWDoTi2Op9pEa0aYhKq1wIxB4B4kowH5RjsskdlXy2Gf2pf5QL4EFVBae8MMwH2INsJc/R90tuIGSwgr+FQ13iD1E3sEGi1VBqAcIRwA99UVUbsDVQc6Li/ENbJuS5JR9wPRqfvr+D5O5n+iRcYbDabo7R+bLIm0IdgAmIvREGmAHmq81Z6txWiJeUIsRZ3BldN9JGnU1fUpmA78TVhfpq42zy6q2EDvjIc91CL0jIJNI2mGljha2T1/gy2vqnIbTDj0zHBVajPEO+26vlUSd9zmDETf4Km3V2tYblUIiDuZwqH2hiiQz4lf1UYqF2sS5XlxcXBBP98YaLHNt0S5N/k1avH/XuaJbPyMyU9ZKE1IXWYTLnoMVnPM+/lD0J9xfeZJ3gLeLtjilARlk6FdzDx5W7IH4u9sdFEktnKq8NXRhkeYXb3CHtSjxYy1PfF6tSOCP9DIaZMcK2Z9Vqivis3+ZbFqxe9vWU92xzRg5ZsPr+AkXHnPOLKvt/SCnjRLT5taJVCBfvOSEXkg3tDKk2DWFczzpHA=
  template:
    metadata:
      annotations: {}
      name: azure-cdtnadmindev-volume
    type: Opaque
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations: {}
  name: azure-cdtnadminprod-volume
  namespace: cdtn-admin
spec:
  encryptedData:
    azurestorageaccountkey: >-
      AgArMu861qm2F7g/QpkOUqz4BAM+AIwuAXsE2PpJMNKKmZz/Pbyg3oIMfdcqg8e3rss7PpddLbZv+TYFCC8HuVIrQ9u5gA1t2HN0qjDGBsDyWr7cF2I+eB71JN287MoOeG90hF+TfkHh3bONZTDEwOT+sYs6XSDm4EsXEE7kkUk0yJjUA0s5uXZYI33dteRWPYMp7KaG73RWl6CW0/woKwnX5ml4d69g8cmAkf345Xe5knqGTh8wt2idJjEnjgF8j3WqCdbD1AUkqnovQ5igr36E/j8+B6GyNZtQVEqSyNZT16o973IpkoOs3ZUk5kAi9TlAxHIdLIUDqmnKr29apu/UGTdOGr0zqUbnJyawh+aCUSzLP3D/CKQFtcGNvs+FqUciOlRt1FoU25GuDMEH7ol3ic5+45jOMeJJNPwhOHxynJvAMQCnJAqGYMQefj4o5LnfRBYpS4CumMGhzuBvfHkVnrYj69d+gJ3YEpJ5aKBu9tCAAgyl9g8KzRTfLPCiSqxKfV7uQoNHilluWdHW8TQssHSzi1+K3WVXhRVTTV0NjQoEt9ZkPb7/iOns8IW5SucSJUABDf7U8C3aG+t0zymjMhpkpsnq/JPK2K4pkO8Qa0R6GSO6YJKVT9sKCYTfD3Mrr3FnROaThouPmQmCFfzVXErgdnAaBJUrG7F5vppguDA+obT9GH13zrurn+bEnvZfgi2oIutV/CMHijYS3tTA0OIEdHZRUb1DHU+efX4YuJc4qF2W117ZgUjv9luxAIFcPRXpkDKeUv1gGYAuKIMatkmLJ6END8iakKgb2NpfsNna8a1MSjAe
    azurestorageaccountname: >-
      AgAfpzqFtomji8nwYM6g5mbXXDAA8REQtfJakS059Vy2KkUbZ1okjVQ5mFN61ODTcBknR/dbncVRsl9n/JoEqLfN1wSp58QOBTFLFXiMNeLACEEIYg/Ko6hZVsUB6xs6x0Ma75ngDr+BcAtUCX9g7qqd4Lpw+7lWNdDrO4yT7BLdbV1WvoHA8YQkSyqZ/w9/lugYwc63mmf5PsUz3JVhHMwN+esOvTOcDqxFwpSR2v3z/o7wgmm3CLSg06LGSK5rAppetk/ojfeWZqWk9nA6It/BQ2x1nlku/2wU4HjO7H1+syPsvftElu4Bw4vcX8SFY5+2oGplwekThIkH/V+YQvsq1mI0y45ZOh8QfwfNJoPNjvEFGXpZX8eKriUAeesMC0y7CRMeSBVXdrmQFe9yjViFsh1gX5G5Jl7bFKMhZORZ/DAvVUfXt4zC/V+y3uo6QrFEp8Ywm/uYO5KwhiGU2B/8gYu3oQUuhjl8M+s+y8jkF+WeCTPe0lsxCYlxQ/BB9e1x86j+ulPtUEvlA4Bp/AMlKUomdLDuX8Mddtp2T85QB+3KuTtBQlWLRqjywhEZjKmourH99JOF2Buh9OX/tGg7S5ZfuUdwHqQh5/dX1S2/VFAJFdFOpt0HEzuiuxTw2l1rw5SYqmZuHRrbVIHgTyxRpUcJXQ+dep6MAuklR42sxynraa620atGW1MdHALd6U90DG8H482QS97v9lH3
  template:
    metadata:
      annotations: {}
      name: azure-cdtnadminprod-volume
    type: Opaque
"
`;
